main/main.cpp

```diff
@@ -259,7 +259,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --resolution <W>x<H>             Request window resolution.\n");
 	OS::get_singleton()->print("  --position <X>,<Y>               Request window position.\n");
 	OS::get_singleton()->print("  --low-dpi                        Force low-DPI mode (macOS and Windows only).\n");
-	OS::get_singleton()->print("  --no-window                      Disable window creation (Windows only). Useful together with --script.\n");
+	OS::get_singleton()->print("  --no-window                      Run with invisible window (X11, Windows anc macOS only). Useful together with --script.\n");
```

Comment:
"anc", it also could use the Oxford comma there as well.

Reply:
I think we can remove the platform-specific note as you can only view the command line help on desktop platforms anyway.

Reply:
Removed!